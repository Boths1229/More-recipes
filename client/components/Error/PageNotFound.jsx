import React from 'react';
import { Link } from 'react-router-dom';

/**
 * @description functional component to render 404 pages
 *
 * @returns {JSX} return JSX
 */
const PageNotFound = () => ((
  <div className="bg-color">
    <div className="container bg-color">
      <div className="section-margin">
        <img className="image-fluid image-position" src="https://www.fg-a.com/faces/not-so-smiley-face.png" alt="" />
      </div>
      <div className="row">
        <div className="col-md-12 text-center pb-2">
          <h1 className="text-font pb-3 pt-3">OOPS!</h1>
          <p className="text-desc pb-5">Page Not Found. That Page Does not Exist!</p>
          <button className="btn btn-lg button-lg">
            <Link to="/recipes"> <span className="btn-white">Go Home</span> </Link>
          </button>
        </div>
      </div>
    </div>
  </div>
));

export default PageNotFound;
