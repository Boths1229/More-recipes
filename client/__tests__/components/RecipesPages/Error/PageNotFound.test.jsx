import React from 'react';
import { shallow } from 'enzyme';
import PageNotFound from '../../../../components/Error/PageNotFound';

describe('Page not found component', () => {
  it('matches snapshot', () => {
    const output = shallow(<PageNotFound />);
    expect(output).toMatchSnapshot();
  });
});
