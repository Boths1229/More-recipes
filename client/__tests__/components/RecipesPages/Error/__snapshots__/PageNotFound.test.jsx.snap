// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page not found component matches snapshot 1`] = `
<div
  className="bg-color"
>
  <div
    className="container bg-color"
  >
    <div
      className="section-margin"
    >
      <img
        alt=""
        className="image-fluid image-position"
        src="https://www.fg-a.com/faces/not-so-smiley-face.png"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="col-md-12 text-center pb-2"
      >
        <h1
          className="text-font pb-3 pt-3"
        >
          OOPS!
        </h1>
        <p
          className="text-desc pb-5"
        >
          Page Not Found. That Page Does not Exist!
        </p>
        <button
          className="btn btn-lg button-lg"
        >
          <Link
            replace={false}
            to="/recipes"
          >
             
            <span
              className="btn-white"
            >
              Go Home
            </span>
             
          </Link>
        </button>
      </div>
    </div>
  </div>
</div>
`;
